### ISSUE TITLE
Warn family if demographic survey is absent

### ISSUE THREAD

**Issue by mekline** [lookit/lookit-api#1144]
**Description**

**How to reproduce**

If you create a new account and click on the "Children Information" button to add a child without completing the demographic form, the "Find a Study" bar stays greyed out and the UI doesn't give any hints about why this might be. 

**Expected behavior**

If the user has deviated from the expected workflow (go through forms one by one, saving and continuing to the next screen) everything is fine, but if you get off track it's not clear how to resume. There should be an indicator of some kind, probably for both the demographic & child, along the lines of "Add your demographic info to see studies you're eligible for", "Add at least one child to see studies you're eligible for".

**Comment by Ester0Awad** [comment]
Hi @becky-gilbert, I'd like to work on this issue next! Could you please assign it to me?

**Comment by becky-gilbert** [comment]
@Ester0Awad yep, that'd be great! But would you mind making the quick requested change to this one first? https://github.com/lookit/lookit-api/pull/1939
